integration_tests/CMakeLists.txt

```diff
@@ -626,6 +626,7 @@ RUN(NAME test_builtin_sum    LABELS cpython llvm c)
 RUN(NAME test_math1          LABELS cpython llvm c)
 RUN(NAME test_math_02        LABELS cpython llvm NOFAST)
 RUN(NAME test_math_03        LABELS llvm)               #1595: TODO: Test using CPython (3.11 recommended)
+RUN(NAME test_math_04        LABELS llvm)               #TODO: Test using CPython (as above)
```

Comment:
Why not enable CPython for this test? Does it fail with CPython?

I think a test should run with CPython so that we are sure that we are supporting and delivering the right syntax and runtime output.

Reply:
Is it because CPython does not have this implemented?
The math module has cbrt(), according to the official python documentation: https://docs.python.org/3/library/math.html.
Is this what I'm supposed to refer to? I couldn't find CPython documentation.

Reply:
I get the same error when I enable CPython for test_math_03 (which has already been merged).
```
Traceback (most recent call last):
  File "/home/kishan/Desktop/lpython/integration_tests/test_math_03.py", line 1, in <module>
    from math import (cbrt, exp2)
ImportError: cannot import name 'cbrt' from 'math' (/home/kishan/anaconda3/envs/lp/lib/python3.10/lib-dynload/math.cpython-310-x86_64-linux-gnu.so)
```

Reply:
It seems `cbrt` in python is supported from version `3.11`. I think we currently use python `3.10` locally and at the CI. Hence, you get the above import error.

I think it is fine for now to not enable the test for CPython.

Reply:
Ok. I face the same issue here too, please check: https://github.com/lcompilers/lpython/pull/2556